lib/kafka/consumer_group.rb

```diff
@@ -186,6 +186,12 @@ def synchronize
       end
     end
 
+    def to_s
+      "[#{@group_id}] {" + assigned_partitions.map { |topic, partitions|
+        "#{topic}: #{partitions.join(', ')}"
+      }.join('; ') + '}:'
```

Comment:
This can quickly grow out of hand – we have topics with 100 partitions (don't ask...)

Reply:
Oof... maybe I can just put in the first 5 and then ... - that should suffice for most use cases.